components/frontend/src/dashboard/LegendCard.jsx

```diff
@@ -10,15 +10,15 @@ export function LegendCard() {
             <StatusIcon status={status} />
             <ListItemText
                 primary={STATUS_SHORT_NAME[status]}
-                slotProps={{ primary: { typography: { fontSize: "12px" } } }}
+                slotProps={{ primary: { typography: { fontSize: "1.1em" } } }}
```

Comment:
Is it correct that this is smaller than FilterCardWithTable's `1.3em` ?

Reply:
Yes, the Legend card has more lines of text.